ui/components/app/snaps/snap-authorship-expanded/snap-authorship-expanded.js

```diff
@@ -53,6 +61,22 @@ const SnapAuthorshipExpanded = ({ snapId, className, snap }) => {
   const subjectMetadata = useSelector((state) =>
     getTargetSubjectMetadata(state, snapId),
   );
+  const snapRegistryData = useSelector((state) =>
+    getSnapRegistryData(state, snapId),
+  );
+  const { website = undefined } = snapRegistryData?.metadata ?? {};
+  const [safeWebsite, setSafeWebsite] = useState('');
+
+  useEffect(() => {
+    const performPhishingCheck = async () => {
+      const phishingResult = await getPhishingResult(website);
+
+      if (!phishingResult.result) {
+        setSafeWebsite(website);
+      }
+    };
+    performPhishingCheck();
```

Comment:
We don't need to do this if the website is undefined.

Reply:
Added check for website before calling.